### PR TITLE
removal of unused annotations

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: osbs/cli/main.py [hunk 1]
```diff
@@ -399,21 +385,13 @@ def cmd_build_logs(args, osbs):
     build_id = args.BUILD_ID[0]
     follow = args.follow
 
-    if follow and args.from_docker_build:
-        print("Can't use --follow and --from-docker-build. "
-              "Logs from docker build are part of metadata of a already built image.")
+    logs = osbs.get_build_logs(build_id, follow=follow,
+                               wait_if_missing=args.wait_if_missing,
+                               decode=True)
```
Comment (lcarva) [inline osbs/cli/main.py]: Any concerns with using `decode=True` if follow if `False` ?

Comment (rcerven) [reply osbs/cli/main.py]: ahh ok I'll put it bellow that if as it should be

Comment (rcerven) [reply osbs/cli/main.py]: mmm this logic wasn't changed, should those 2 things be conflicting?

Comment (lcarva) [reply osbs/cli/main.py]: Yes, you're right. Logic is the same.

I vaguely remember a while back there was an issue with follow and decode.

**Comment by rcerven** [comment]
rn updated:

## API changes
- Removal of get_docker_build_logs method 

## Improvements
- Removal of methods related to logs and rpm-packages annotations

**Pull request merged**